students/erik_oien/lesson09/mailroom_oo/donor_models.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+import os
+
+class Donor():
+
+    def __init__(self, name, donation_amount=None):
+        self.name = name
+        if donation_amount is None:
+            self.donation_amount = []
+        else:
+            self.donation_amount = donation_amount
+
+    def add_donation(self, amount):
+        self.donation_amount.append(amount)
+
+    def thank_you_letter(self):
+        thank_you = "\n".join(("Dear {},\n".format(self.name),
+        "Thank you for your generous donation of ${}! As you certianly know, kittens".format(self.donation_amount[-1]),
+        "and metal are awesome and your donation will insure that others will be able",
+        "to enjoy kittens and metal.\n",
+        "Thank you,\n",
+        "Kittens and Metal Charity\n"))
+        return thank_you
+
+class DonorCollection():
+    
+    def __init__(self):
+        self.donors = {}
+    
+    def initialize_donor_dict(donor_data):
```

Comment:
this should be either a staticmethod or classmethod - the way it is defined is not correct.